content/md/retailer-onboarder/suppliers-management/supplier-definition.md

```diff
@@ -18,5 +18,6 @@ A supplier is one of your business partner. You buy him products to expand your
 | UI locale | The language of the Supplier Onboarder interface | Among the PIM locales |
 | Catalog locale | The locale used in the supplier Onboarder for product data | Among the activated PIM locales |
 | Families | The families the supplier can work with |
+| Allow product creation | Enable suppliers to create or not new products | |
```

Comment:
Enable suppliers to create or not new products --> Allow the supplier to create new products or not

Reply:
Maybe also talk about the default value.

Reply:
And what about flat files imports? Are there any changes?